### DISCUSSION TITLE
Confusion over comparing tags across forks

### DISCUSSION THREAD

**Discussion by graysky2** [community/community#22881] [New to GitHub]
<p>I’d like to compare tags across forks in via the URL method but am unable to see diffs (clealy doing this incorrectly).  My goal is to see how archlinux/linux tag=v4.17.13-arch1 differs from linux/torvalds  tag=v4.17.13 (ie to see what the Arch devs are patching their version of the kernel with relative to the vanilla release).</p>
<p>I thought the following would do this but it does not give any diffs: <a href="https://github.com/archlinux/linux/compare/v4.17.13-arch1...torvalds:v4.17.13" rel="nofollow noopener">https://github.com/archlinux/linux/compare/v4.17.13-arch1…torvalds:v4.17.13</a></p>
<p>I’m not sure how to fold in the stable tags to Linus’ tree in the url… on the shell the following works:</p>
<pre><code>git clone https://github.com/archlinux/linux.git
cd linux
git remote add stable https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
git fetch --tags stable
git log v4.17.13..v4.17.13-arch1
</code></pre>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/graysky2">@graysky2</a>,</p>
<p>Thanks so much for being part of the GitHub Community Forum. I’ll do all I can to help.</p>
<p>First, it looks like you are trying to compare using a tag that doesn’t actually exist. <code>torvalds</code> actually doesn’t appear to have a <code>v4.17.13</code> tag, but only a <code>v4.17</code> tag. Fixing that reference should help.</p>
<p>Additionally, you’ll want to switch the order of the URL for better results. Instead of setting up the URL so that you’re querying <code>archlinux:v4.17.13-arch1</code> --&gt; <code>torvalds:v4.17</code>, you’ll want to do this:</p>
<p><a href="https://github.com/torvalds/linux/compare/v4.17...archlinux:v4.17.13-arch1" rel="nofollow noopener">https://github.com/torvalds/linux/compare/v4.17…archlinux:v4.17.13-arch1</a></p>
<p>That should yield better results for you.</p>
<p>I hope this helps! Let me know if you have further questions.</p>

**Comment by graysky2** [comment, marked as answer]
<p><a class="mention" href="/nadiajoyce">@nadiajoyce</a> - Thank you for the thoughtful reply.  I acutally parallel posted here and on the <a href="https://bbs.archlinux.org/viewtopic.php?pid=1801268#p1801268" rel="nofollow noopener">Arch linux forums</a>.  A user there (progandy) suggested the following which seems to work very well.</p>
<pre><code>tag=v4.17.13
archtag=v4.17.13-arch1
tagged="$(curl "https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tag/?h=$tag" | sed -n 's#.*/linux.git.commit.*id=\(\w*\).*#\1#p')"
xdg-open "https://github.com/archlinux/linux/compare/$tagged...$archtag"
</code></pre>
